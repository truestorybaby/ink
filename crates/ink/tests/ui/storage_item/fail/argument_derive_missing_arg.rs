#[ink::storage_item(derive)]
#[derive(Default)]
struct Contract<KEY: StorageKey = ManualKey<123>> {
    a: u16,
    b: u64,
    c: u128,
}

fn main() {}
