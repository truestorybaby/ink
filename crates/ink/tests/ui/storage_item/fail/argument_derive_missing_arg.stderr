error: ink! config options require an argument separated by '='
 --> tests/ui/storage_item/fail/argument_derive_missing_arg.rs:1:21
  |
1 | #[ink::storage_item(derive)]
  |                     ^^^^^^
